osd_cluster_admin/osd_hibernate_resume/osd-overview-hibernating-resuming-cluster.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+// * osd_cluster_admin/osd_hibernate_resume/osd-overview-hibernating-resuming-cluster.adoc
+
+:_content-type: CONCEPT
+[id="osd-overview-hibernating-resuming-cluster_{context}"]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+include::_attributes/common-attributes.adoc[]
+= Overview of hibernating and resuming cluster (Technology Preview)
+
+:context: osd-overview-hibernating-resuming-cluster
+
+toc::[]
+
+As a cluster administrator, you might need to hibernate a {product-title} cluster for a period and resume it later to conserve resources.
+
+[IMPORTANT]
+====
+Hibernating a ready cluster and resuming a hibernating cluster are Technology Preview features only. The hibernate and resume features for {product-title} clusters are provided as Technology Preview features as explained in link:https://access.redhat.com/articles/7012966[Technical Preview: Hibernating and Resuming Clusters]. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. 
+
+During the entire period of hibernation and till the resumed cluster is ready, the cluster is in link:https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-limited-support_rosa-service-definition[Limited Support] status under {product-title} SLA.  For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]
+====
+
+A hibernating cluster requires significantly fewer resources than one that is running, potentially leading to lowered provider costs through the strategic movement of clusters in and out of a hibernating state. Hibernating a ready cluster on-demand can notably reduce infrastructure expenses, including those related to EC2 instances and network devices. 
+
+This approach is particularly effective for development or test clusters that are only intermittently used by end users. However, it is important to note that hibernation does not lead to cost savings in all aspects. While it helps curtail expenses for items such as EC2 instances and network devices, expenses tied to EBS and S3 continue to accrue during the hibernation period. This is particularly relevant when you have an annual commitment or a prepaid quota in place, as these costs are not mitigated by hibernation. 
```

Comment:
We must say that:
1. Hibernation is an useful tool for non-critical clusters that does not contain any non-recoverable business data or application services. 
2. Hibernation is recommended where other cost-saving features like auto-scaling or complete recreation of clusters through automation are not applicable.
3. Hibernation saves on Amazon EC2 costs but the cluster may incur additional costs from services such as Amazon EBS and S3.
4. Red Hat Subscription will continue to accrue during the time cluster is hibernated.

Reply:
Updated Line 25 as, "Hibernation is a useful tool for non-critical clusters that do not contain any non-recoverable business data or application services. Hibernation is recommended where other cost-saving features like auto-scaling or complete recreation of clusters through automation are not applicable. Hibernation saves on Amazon EC2 costs, but the cluster may incur additional costs from services such as Amazon EBS and S3. Red Hat subscription continue to accrue during the time the cluster is hibernated."

Reply:
I believe we can remove Line 27 now. @bchandra-ocp Would like to hear your thoughts. Thanks!

Reply:
Removed Line 27. @bchandra-ocp Please let me know if you have other thoughts.